src/dune/coq_rules.ml

```diff
@@ -422,7 +426,7 @@ let extract_rules ~sctx ~build_dir ~dir ~dir_contents (s : Extract.t) =
   in
   let coqc =
     let open Build.O in
-    coqc_rule cctx ~build_dir ~file_flags coq_module
+    coqc_rule cctx ~file_flags coq_module
     |> Build.With_targets.map_build ~f:(fun build -> cctx.mlpack_rule >>> build)
```

Comment:
I'm unsure why the mlpack_rule is here, this is something only coqdep needs.

Reply:
I got rid of it.